proposals/0216-widget-support.md

```diff
@@ -52,13 +52,13 @@ Today in the vehicle, SDL works as a system with a single main display and a sin
 
 #### System
 
-The terminology "system" in this proposal means the system's endpoint running inside the machine where the app connects to. In the classical term this means SDL core running in IVI.
+The terminology "system" in this proposal means the system's endpoint running inside the machine where the app connects to. In classical terms this means SDL core running in IVI.
```

Comment:
I noted theses changes in my first review, do you disagree with them? 

The terminology "system" in this proposal means the system's endpoint **is** running inside the machine where the app connects to. In classical terms this means SDL core **is** running in IVI.

Reply:
I'm sorry. I only saw "classical terms" as bold text. So in the text I want to say "system" is used as a term for some object/instance, not for a status/behavior. I changed the text to 

> The terminology "system" in this proposal means the system's **endpoint, that is** running inside the machine where the app connects to. In classical terms this means **SDL core, that is** running in IVI.